frontend/src/hooks/api/actions/useMessageBannersApi/useMessageBannersApi.ts

```diff
@@ -0,0 +1,68 @@
+import { IInternalMessageBanner } from 'interfaces/messageBanner';
+import useAPI from '../useApi/useApi';
+
+const ENDPOINT = 'api/admin/message-banners';
```

Comment:
Just wondering, should we just call them "banners" instead of "message-banners"? I feel we might regret in the future for not giving it a more general name

Reply:
Well, you know more than I do, so I trust your judgment. It was just an observation because message-banner seems weird... a banner is by definition a message you want to deliver... the fact that it's a "message banner" type is an implementation detail IMO. But again, take it for what it is: opinions without a lot of context :D